brick/hooks/pre_gen.dart

```diff
@@ -1,5 +1,27 @@
 import 'package:mason/mason.dart';
+import 'package:recase/recase.dart';
+
+enum Platform {
+  android,
+  ios,
+}
 
 void run(HookContext context) {
   context.vars['current_year'] = DateTime.now().year;
+
+  context.vars['application_id_android'] =
+      _appId(context, platform: Platform.android);
+  context.vars['application_id_ios'] = _appId(context, platform: Platform.ios);
```

Comment:
Considering this is also used for windows, I would opt for a name that reflects that a bit better, or split it up in it's own variable